integrations/access/accessrequest/message.go

```diff
@@ -80,11 +81,22 @@ func MsgFields(reqID string, reqData pd.AccessRequestData, clusterName string, w
 	if len(reqData.User) > 0 {
 		msgFieldToBuilder(&builder, "User", reqData.User)
 	}
-	if len(reqData.Roles) > 0 {
-		msgFieldToBuilder(&builder, "Role(s)", strings.Join(reqData.Roles, ","))
+	if len(reqData.LoginsByRole) > 0 {
+		for role, logins := range reqData.LoginsByRole {
```

Comment:
I think you can drop the `if len(....) ` because the `for range` will not iterate if the len is 0.

Reply:
The way it's implemented the function will either build the message with `LoginsByRole` or `Roles`. This allows us to maintain the previous behavior if logins are not provided. We could get rid of this if statement, but we'd then need to verify that `len(LoginsByRole) == 0` before adding the `Roles`.

Once we've made this change to using `LoginsByRole` for all plugins, we can then remove this `Roles` field and if statement.